index.html

```diff
@@ -1504,6 +1504,47 @@ <h2>Public Keys</h2>
 see Section <a href="#resolution"></a>.
       </p>
 
+<p>
+Revoked keys MUST NOT be present in the <code>publicKey</code> property of a <a>DID document</a>.<br/>
+Revoked keys MAY be present in the <code>revokedKey</code> property of a <a>DID document</a>.
```

Comment:
-1, this is the wrong thing to do. I think `publicKey` was a mistake and was ok agreeing to it as a compromise. Doubling down on a bad decision and adding `revokedKey` is a realization of exactly what I thought was going to happen if we went down the `publicKey` slippery slope.

There are more than just keys in DID Documents... `publicKeys` are just a special case bucket for `verificationMethod`s. We added it a long time ago because people wanted a single bucket to put keys into that they could refer to from elsewhere in the document. What we really should have done if we wanted a bucket (which was a tenuous compromise to start) is specify a bucket called `verificationMethod`. 

So, -1 to revokedKey because we don't have a strong driving use case for the property.

Reply:
I agree with your intention, but I chose this specifically so that people would not need to process `publicKey`, or other verificationMethods differently to understand if things were revoked.

I don't want revokedKeys to be in a did document period... @jandrieu I think you wanted them in the did document.

If we are going to have them in a did document... I want them to be ONLY in one place... like `revokedKey`.

Reply:
As I have revised this content, I request to resolve this thread to avoid confusion.